src/mailEclipse.php

```diff
@@ -686,7 +686,7 @@ private static function getMailableViewData($mailable, $mailable_data)
         $allProps = [];
 
         foreach ($properties as $prop) {
-            if ($prop->class == $data->getName()) {
+            if ($prop->class == $data->getName() || $prop->class == get_parent_class($data->getName()) && !$prop->isStatic()) {
                 $allProps[] = $prop->name;
             }
```

Comment:
Maybe you could structure the logic like the below, it could be a bit more readable in the future and not confusing to new contributors with multiple logic operators in a single lined if statement. 

```suggestion
            if (
                $prop->class == $data->getName()
                || $prop->class == get_parent_class($data->getName()) && !$prop->isStatic()
            ) {
                $allProps[] = $prop->name;
            }
```

It is just a style request, but it does convey the intention further.